### ISSUE TITLE
Código de Conduta Convergente

### ISSUE THREAD

**Issue by antonioribeiro** [convergencias/selo-convergente#11]
# Código de Conduta Convergente

Todas as discussões e busca pela Convergência devem ser pautadas pela civilidade e a entrega de dados necessários ao diálogo, a orientação pessoal de cada um deve servir como argumento, mas não como arma ou ferramenta para ataque a membros do grupo.

### O que devemos entregar numa discussão que busca Convergência?

- Cordialidade
- Fraternidade / sororidade
- Ideias
- Argumentos
- Fatos 
- Dados e fontes dos mesmos
- Links das matérias
- Metodologias
- Sempre valorizar o conhecimento científico

### Condutas **NÃO ADMITIDAS**

- Agressões diretas ou falta de respeito a membros do grupo
- Uso de "Argumentum ad hominem" (argumentos contra o autor e não contra o assunto)
- Usar **CAPS LOCK**
- Truculência nas colocações
- Palavrões **excessivos**
- Fake News
- Sarcasmo
- Excessos no envio de áudios
- Abuso de poder (dos moderadores)

### Deve-se evitar

- Generalizações
- Falácias e saltos lógicos 
- Matérias e links repetidos
- Participar de uma discussão sem ter conhecimento do que já foi dito anteriormente
- Mudar o assunto corrente da discussão (evitar atravessar a conversa)
- Cristalização da opinião
- Falta de clareza do ponto a convergir

### Sexta sem lei

- Vale (quase) tudo, desde que seja para fazer rir
- Use a hashtag #sextasemlei

### Moderação

Para manter um nível razoável de civilidade, em casos extremos**, os moderadores do grupo podem ter que atuar e isso pode significar:

- Pedidos de "baixe o tom, por favor"
- Advertências
- Exclusão **temporária** de um ou mais membros do grupo
- Perda do status de moderador
- Silenciamento temporário do grupo todo (circuit breaker)

**Comment by Coutada** [comment]
Creio que devemos ter um colegiado (de três pessoas ao menos e sempre ímpar) para que decisões sejam tomadas de forma que não gere uma grande carga de responsabilidades para uma única pessoa.
Além de despersonalizar decisões.

**Comment by lecouto** [comment]
@Coutada 3 eu acho pouco pro numero de pessoas do grupo. Acho que 5 poderia ser mais representativo e não seria tão administrativamente custoso pra decidir.

**Comment by antonioribeiro** [comment]
O problema é que poucas pessoas são atuantes de verdade, de 145 apenas umas 10 ou 20 estão ativas de verdade

**Comment by Coutada** [comment]
@lecouto concordo e creio que poderia até ser maior mas acho que já será difícil arrumar 3 pra tarefa considerando que a Juliet continue fazendo o ótimo trabalho de moderação que já realiza. Ou seja teríamos que encontrar pelo menos mais 2.
